tide_core.install

```diff
@@ -225,3 +225,17 @@ function tide_core_update_8006() {
     }
   }
 }
+
```

Comment:
@vincent-gao do we need update hooks to disable the module (Will it just remove the tables in this case, so the update below not needed) and a future ticket to remove from `composer.json`? Also, should we be using the field API to remove the fields that are marked as removed in the yml files?

Reply:
The schduled_updats configs are spread across tide modules. and all of them refer to the tide_test and tide_core module. this is why I dealt with `tide_test` and `tide_core` first.

here is the plan:
1. the update hook for deleting and uninstalling scheduled_updates module will be in tide_core
2. delete all of the scheduled_updates configs from other tide modules like tide_page, tide_landing_page ...

We don't need to use field API to remove the fields I think. after `schduled_updats` module gets removed from all dependencies, we will have an installation without schduled_updats.

Reply:
the test failed, but I think this is c.

Reply:
@vincent-gao Scheduled Updates fields of all content types depend on the `scheduled_updates.scheduled_update_type.*` Scheduled Update Types. Those Types are entities defined by the Scheduled Updates module.

From my understanding, you cannot uninstall a module if there are existing entities defined by that module.

Reply:
@sonnykt yes. that is right. we have an update_hook for deleting all existing entities(only 2 scheduled entities).

Reply:
@sonnykt after this gets merged to develop, I will delete dependencies and configs from other tide submodules. finally dealing with client sites like content-vic, actually content-vic is already modified by https://github.com/dpc-sdp/content-vic-gov-au/pull/616